src/Hl7.Fhir.Support.Poco/Rest/IRequester.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Hl7.Fhir.Model;
+using System;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Hl7.Fhir.Rest
+{
+    public interface IRequester
```

Comment:
Interesting. If we don't need this for testing, then I'd like to remove as much as possible. Or make it internal.

Reply:
We use is in the `BaseFhirClient` so we can't remove it or make it internal

Reply:
Ah, it is used as a trick to make both flavours of the requestor work within the single BaseFhirClient! ;-)

Good, but maybe then we should rename the interface to something more precise - IRequester sounds like something we'd like to use in another context for totally different purposes in the future.  Something like IHttpRequester or IEntryRequester?